01-Introduction and Goals.md

```diff
@@ -4,70 +4,24 @@
 
 ## Quality Goals
 
-* Entity data which is included in a client wiki's article content is eventually consistent with updates on the corresponding repo. Data updates from the repo are available to the client within [10 minutes](https://gerrit.wikimedia.org/r/plugins/gitiles/operations/puppet/+/e3423c8627067a8de8352b9b7f3c59870d9b47c5/modules/icinga/manifests/monitor/wikidata.pp#10).
-
 ## Stakeholders & Expectations
 
-### Wikidata editor
-
-- Add new structured entity data on Wikidata via a desktop web interface.
-- Edit existing structured entity data on Wikidata via a desktop web interface.
-- Programmatically add new structured data to Wikidata.
-- Programmatically edit existing structured data on Wikidata.
-- View existing structured entity data on Wikidata on mobile web.
-- Edit labels, descriptions, aliases on Wikidata via a mobile web interface.
-- Edit terms and sitelinks of entities without using JavaScript in a browser.
-- Have Statements about external identifiers separated from other Statements in the Wikidata UI.
-- Have Statements shown in a useful order.
-- Search on Wikidata for an entity.
-- Ensure the coherence of data modeling in a certain topic.
-- Review edits.
-
-### Wikipedia editor
-
-- Connect pages on different Wikipedias, showing interwiki links on the Articles.
-- Store Wikipedia page connections in a central place.
-- Store infomation relating to article quality in a central place, for use in interwiki links on the Articles.
-- Use data from Wikidata in Wikipedia pages.
-- Have Wikipedia pages automatically update when data on Wikidata changes.
-- Have data updates which cause page changes to be shown in RecentChanges and Watchlists.
-- Update Wikidata’s data used on Wikipedia when it is incorrect or outdated via web interface.
-- See the "unconnected pages" on a Wikipedia to try to create connections.
-- Find wrong or missing data on a Wikipedia article and correct or add it to Wikidata.
-
-#### Small Wikipedia editor
-
-- Have placeholder pages be automatically generated for topics Wikidata contains data for, in order to gain more readers and thus more editors.
-
-### Commons editors
+Stakeholders and expeectations are slowly being sorted into sub component templates.
 
-- Store structured data relevant to files on Commons, on Commons.
-- Describe Commons files using concepts from Wikidata.
+Some stakeholders and expectations may stay at this high level though.
 
-### Data re-user
+### Small Wikipedia editor
 
-- Access Wikidata's content via a web API.
-- Fix and add values on Wikidata.
+- Have placeholder pages be automatically generated for topics Wikidata contains data for, in order to gain more readers and thus more editors.
 
-#### Researcher
+### Wikidata Researcher
```

Comment:
I liked just "Researcher" better than "Wikidata Researcher". The latter sounds like a person doing research about Wikidata to me?

Reply:
Will change this back!